packages/plugin-ext/src/main/browser/plugin-contribution-handler.ts

```diff
@@ -140,6 +145,11 @@ export class PluginContributionHandler {
                     });
                     monaco.languages.onLanguage(grammar.language, () => this.monacoTextmateService.activateLanguage(grammar.language!));
                 }
+                for (const editor of MonacoEditor.getAll(this.editorManager)) {
```

Comment:
[question]: Does it have to run for each `grammar`? Cannot we move outside of the `for`?

Reply:
good point!